src/wp-includes/cache-compat.php

```diff
@@ -141,3 +141,30 @@ function wp_cache_flush_runtime() {
 		return wp_using_ext_object_cache() ? false : wp_cache_flush();
 	}
 endif;
+
+if ( ! function_exists( 'wp_cache_flush_group' ) ) :
+	/**
+	 * Removes all cache items in a group, if the object cache implementation supports it.
+	 * Before calling this method, always check for group flushing support using the
+	 * `WP_OBJECT_CACHE_SUPPORTS_GROUP_FLUSH` constant.
+	 *
+	 * @since 6.0.0
+	 *
+	 * @see WP_Object_Cache::flush_group()
+	 * @global WP_Object_Cache $wp_object_cache Object cache global instance.
+	 *
+	 * @param string|array $group name(s) of group to remove from cache.
+	 * @return bool|array|WP_Error Bool or array of bool if array passed, WP_Error if not supported.
+	 */
+	function wp_cache_flush_group( $group ) {
+		if ( defined( 'WP_OBJECT_CACHE_SUPPORTS_GROUP_FLUSH' ) && WP_OBJECT_CACHE_SUPPORTS_GROUP_FLUSH ) {
+			return $wp_object_cache->flush_group( $group );
+		}
+
+		$error = new WP_Error( 'unsupported', __( 'Your object cache implementation does not support flushing individual groups.' ) );
+
+		_doing_it_wrong( __FUNCTION__, $error->get_error_message(), '6.0.0' );
+
+		return $error;
```

Comment:
Swap around for readability.

```suggestion
		if ( ! defined( 'WP_OBJECT_CACHE_SUPPORTS_GROUP_FLUSH' ) || ! WP_OBJECT_CACHE_SUPPORTS_GROUP_FLUSH ) {
            $error = new WP_Error( 'unsupported', __( 'Your object cache implementation does not support flushing individual groups.' ) );

		    _doing_it_wrong( __FUNCTION__, $error->get_error_message(), '6.0.0' );

		    return $error;			
		}

		return $wp_object_cache->flush_group( $group );
```